python_modules/libraries/dagster-dg/dagster_dg/config.py

```diff
@@ -226,7 +227,35 @@ class DgRawCliConfig(TypedDict, total=False):
 # ##### PROJECT
 # ########################
 
-DgProjectPythonEnvironment: TypeAlias = Literal["active", "persistent_uv"]
+DgProjectPythonEnvironmentFlag = Literal["active", "uv_managed"]
+
+
+@dataclass
+class DgProjectPythonEnvironment:
+    active: bool = False
+    path: Optional[str] = None
+    uv_managed: bool = False
```

Comment:
Defaults here very dangerous. We should be explicit at callsites

Reply:
I still don't like it and would prefer explicit kwarg at all callsites. However I will not block this.